### DISCUSSION TITLE
Which version of Postgres is compatible with Kratos?

### DISCUSSION THREAD

**Discussion by sergio101** [ory/kratos#1790] [Q&A] [answered]
I am working on getting Kratos working as follows:

We have a managed (AWS) Postgres database (version 12).
I would like to have Kratos connect to that database.
When connecting to that database, I get the error: 

```
_logout_token.sqlite3.down.sql because dialect is not supported: unsupported dialect sqlite3 audience=application service_name=Ory Kratos service_version=v0.7.3-alpha.1
```

I tried connecting to my local machine, and updating the dockerfile to reference:

```
  postgresd:
    image: postgres:12.8
```

and get the same error, so I suspect the issue is with the Postgres version.

The only mention I see in the documentation is here: https://www.ory.sh/keto/docs/configure-deploy/

Where it says: `The SQL adapter supports two DBMS: PostgreSQL 9.6+`

Does the `+` mean 9.6 and greater.. or 9 and anything greater than .6?

Thanks!

**Comment by aeneasr** [comment, marked as answer]
You can ignore this error, it's just a debug message

**Reply by sergio101** [reply]
Thanks! Looking at this further, this isn't my issue as much as the issue is.. i need to connect to postgres running on my local machine from inside my docker image.
